src/Pact/Analyze/Types/ObjUtil.hs

```diff
@@ -12,6 +12,9 @@
 {-# LANGUAGE TypeFamilies         #-}
 {-# LANGUAGE TypeOperators        #-}
 {-# LANGUAGE UndecidableInstances #-}
+
+-- | Type definitions and utilities for manipulating 'HList's and singletons for
```

Comment:
suggestion: "for building 'HList's and object schema singletons.

Reply:
ty